lib/services/route53zone/index.js

```diff
@@ -0,0 +1,141 @@
+ * See https://github.com/byu-oit-appdev/handel/wiki/Creating-a-New-Service-Deployer#service-deployer-contract
+ *   for contract method documentation
+ */
+
+exports.check = function (serviceContext) {
+    let errors = [];
+
+    let params = serviceContext.params;
+
+    if (!params.name) {
+        errors.push(`${SERVICE_NAME} - 'name' parameter must be specified`);
+    } else if (!route53.isValidHostname(params.name)) {
+        errors.push(`${SERVICE_NAME} - 'name' parameter must be a valid hostname`);
+    }
+
+    if (params.private && typeof params.private !== 'boolean') {
+        errors.push(`${SERVICE_NAME} - 'private' parameter must be 'true' or 'false'`);
+    }
+
+    return errors;
+}
+
+exports.preDeploy = function (serviceContext) {
+    return preDeployPhaseCommon.preDeployNotRequired(serviceContext, SERVICE_NAME);
+}
+
+exports.bind = function (ownServiceContext, ownPreDeployContext, dependentOfServiceContext, dependentOfPreDeployContext) {
+    return bindPhaseCommon.bindNotRequired(ownServiceContext, dependentOfServiceContext, SERVICE_NAME);
+}
+
+exports.deploy = function (ownServiceContext, ownPreDeployContext, dependenciesDeployContexts) {
+    let stackName = deployPhaseCommon.getResourceName(ownServiceContext);
+    winston.info(`${SERVICE_NAME} - Deploying Route53 Zone ${stackName}`);
+
+    return getCompiledRoute53Template(ownServiceContext)
+        .then(compiledTemplate => {
+            let stackTags = deployPhaseCommon.getTags(ownServiceContext);
+            return deployPhaseCommon.deployCloudFormationStack(stackName, compiledTemplate, [], true, SERVICE_NAME, stackTags);
+        })
+        .then(createdOrUpdatedStack => {
+            winston.info(`${SERVICE_NAME} - Finished deploying S3 bucket ${stackName}`);
+            return getDeployContext(ownServiceContext, createdOrUpdatedStack);
+        });
+}
+
+exports.consumeEvents = function (ownServiceContext, ownDeployContext, producerServiceContext, producerDeployContext) {
+    return Promise.reject(new Error(`The ${SERVICE_NAME} service doesn't consume events from other services`));
+}
+
+exports.produceEvents = function (ownServiceContext, ownDeployContext, consumerServiceContext, consumerDeployContext) {
+    return Promise.reject(new Error(`The ${SERVICE_NAME} service doesn't currently produce events for other services`));
+}
+
+exports.unPreDeploy = function (ownServiceContext) {
+    return deletePhasesCommon.unPreDeployNotRequired(ownServiceContext, SERVICE_NAME);
+}
+
+exports.unBind = function (ownServiceContext) {
+    return deletePhasesCommon.unBindNotRequired(ownServiceContext, SERVICE_NAME);
+}
+
+exports.unDeploy = function (ownServiceContext) {
+    return deletePhasesCommon.unDeployService(ownServiceContext, SERVICE_NAME);
+}
+
+exports.producedEventsSupportedServices = []; //TODO - No events supported yet, but we will support some like Lambda
+
+exports.producedDeployOutputTypes = [
+    'environmentVariables',
+    'policies'
```

Comment:
This one doesn't output any policies for consumers, correct? If so, let's remove 'policies' from this list, because this list is used when determining which services can be dependencies of each other.